responsibleai/text/components/rai_text_insights/spec.yaml

```diff
@@ -31,24 +31,28 @@ inputs:
   enable_error_analysis:
     type: boolean
     default: True
+  use_model_dependency:
+    type: boolean
+    default: False
 outputs:
   dashboard:
     type: path
   ux_json:
     type: path
-code: ../src/
-environment: azureml://registries/azureml/environments/responsibleai-text-ubuntu20.04-py38-cpu/versions/1
+code: ../src
+environment: azureml://registries/azureml/environments/responsibleai-text-ubuntu20.04-py38-cpu/versions/10
```

Comment:
To confirm version is moving from 1 to 10?

Reply:
yep to the next environment release